rest/rest-server/src/main/java/org/ow2/proactive_grid_cloud_portal/rm/RMStateCaching.java

```diff
@@ -113,4 +114,21 @@ public static RMStateDelta getRMInitialState(long counter) {
             throw e;
         }
     }
+
+    public static RMStateFull getRMStateFull() {
+        try {
+            long startTime = System.currentTimeMillis();
+
+            final RMStateFull state = PAFuture.getFutureValue(rm.getRMStateFull());
+
+            long time = System.currentTimeMillis() - startTime;
```

Comment:
You could put this computation in a `if (logger.isDebugEnabled())`, together with the log.

Reply:
I do not like this logger.is$LOG_LEVEL$Enabled() pattern. But

Reply:
That could avoid the time computation otherwise. Not a big win I agree.